doc/devel/development_workflow.rst

```diff
@@ -415,9 +415,9 @@ GitHub also adds the helpful suggestion to pull the remote changes and then try
 pushing again.  In some cases, such as if you and a colleague are both
 committing and pushing to the same branch, this is a correct course of action.
 
-However, in the case of having intentionally re-written history we *want* to
+However, in the case of having intentionally re-written history, we *want* to
 discard the commits on the remote and replace them with the new-and-improved
-versions from our local branch.  In this case, what we want to do is ::
+versions from our local branches.  In this case, what we want to do is ::
```

Comment:
We're only talking about a single branch.
```suggestion
versions from our local branch.  In this case, what we want to do is ::
```

Reply:
Thank you for your help!